docs/specs/app2app.md

```diff
@@ -111,6 +111,14 @@ Assume there are two apps, A and B. App A is holding a valid user session, and a
 
 #### Android SDK
 
+The following field will be added to `AuthenticateOptions`:
+
+- `app2appOptions: App2AppOptions?`
+  - App2app options. If `null`, this app cannot authenticate other apps through app2app.
+  - `App2AppOptions` contains the following fields:
+    - `userAuthenticationRequired: Boolean`: Whether the user has to pass an authentiction process during the app2app flow. Read [this doc](<https://developer.android.com/reference/android/security/keystore/KeyGenParameterSpec.Builder#setUserAuthenticationRequired(boolean)>) for details.
+    - `userAuthenticationTypes: Int?`: Type of user authentication can be used. Read [this doc](<https://developer.android.com/reference/android/security/keystore/KeyGenParameterSpec.Builder#setUserAuthenticationParameters(int,%20int)>) for details.
```

Comment:
We should follow the existing option we have for biometric. See BiometricAccessConstraintAndroid and BiometricOptionsAndroid in the react native SDK.